ee/hogai/eval/ci/eval_root_documentation.py

```diff
@@ -0,0 +1,365 @@
+                    id="call_doc_search_25",
+                ),
+            ),
+            EvalCase(
+                input="PostHog webhook integration with Slack how to set up",
+                expected=AssistantToolCall(
+                    name="search",
+                    args={"kind": "docs", "query": "webhook Slack integration setup"},
+                    id="call_doc_search_26",
+                ),
+            ),
+            EvalCase(
+                input="can posthog integrate with segment?",
+                expected=AssistantToolCall(
+                    name="search",
+                    args={"kind": "docs", "query": "segment integration"},
+                    id="call_doc_search_27",
+                ),
+            ),
+            # Performance and limits
+            EvalCase(
+                input="What are PostHog's rate limits for event ingestion?",
+                expected=AssistantToolCall(
+                    name="search",
+                    args={"kind": "docs", "query": "rate limits event ingestion"},
+                    id="call_doc_search_28",
+                ),
+            ),
+            EvalCase(
+                input="my posthog is slow how to optimize performance",
+                expected=AssistantToolCall(
+                    name="search",
+                    args={"kind": "docs", "query": "optimize performance"},
+                    id="call_doc_search_29",
+                ),
+            ),
+            # Mobile and platform-specific
+            EvalCase(
+                input="PostHog iOS SDK setup guide",
+                expected=AssistantToolCall(
+                    name="search",
+                    args={"kind": "docs", "query": "iOS SDK setup"},
+                    id="call_doc_search_30",
+                ),
+            ),
+            EvalCase(
+                input="android posthog tracking not working",
+                expected=AssistantToolCall(
+                    name="search",
+                    args={"kind": "docs", "query": "android tracking not working troubleshooting"},
+                    id="call_doc_search_31",
+                ),
+            ),
+            EvalCase(
+                input="flutter posthog plugin how to use",
+                expected=AssistantToolCall(
+                    name="search",
+                    args={"kind": "docs", "query": "flutter plugin usage"},
+                    id="call_doc_search_32",
+                ),
+            ),
+            # Ensure calls docs, not insights
+            EvalCase(
+                input="Is there a field on a person I can use to show the last time they interacted with the platform?",
+                expected=AssistantToolCall(
+                    name="search",
+                    args={"kind": "docs", "query": "person field last interaction time"},
+                    id="call_doc_search_33",
+                ),
+            ),
```

Comment:
This case is no longer relevant, as actually the right tool for "Is there a field" is the taxonomy reading one